src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components/macro';
+import home from './Home.svg';
+
+function Dashboard({ letActiveUser }) {
+  return (
+    <Wrapper>
+        <HeaderWrapper>
+            <img src={home} alt="Menu"></img>
+        </HeaderWrapper>
+        <BodyWrapper>
+            <BigHeadline>Guten Abend, <br/><span>{letActiveUser}!</span></BigHeadline>
```

Comment:
I would suggest trying to keep away from `br` elements and do this via styling.
```suggestion
            <BigHeadline>Guten Abend, <br/><span>{activeUser}!</span></BigHeadline>
```